### ISSUE TITLE
Scanning a product do not record the portion amount. (Not 100g)

### ISSUE THREAD

**Issue by Dalilali** [simonoppowa/OpenNutriTracker#94]
When scanning a product, the application correctly identifies and records the values per portion but incorrectly lists 100g as the portion value.

![WhatsApp Image 2024-06-15 at 18 52 17_1b063e2e](https://github.com/simonoppowa/OpenNutriTracker/assets/102436444/b97e093b-8e61-4342-a2d8-c5cb3e05d99b)
![WhatsApp Image 2024-06-15 at 18 52 43_dec3a0ed](https://github.com/simonoppowa/OpenNutriTracker/assets/102436444/47726ec6-ec8f-447e-9fd1-78a402f2db8a)

**Comment by LesikEdelweiss** [comment]
It could be the problem in underlying "OpenFoodFacts" service data. Maybe someone has entered it wrong

**Comment by TomAFrench** [comment]
Yep, we're just scanning the barcode so portion sides are coming from OpenFoodFacts. If you update the portion size there then the app will pick up the new portion size.
